app/scripts/templates/index.mustache

```diff
@@ -0,0 +1,34 @@
+<div id="main-content" class="card">
```

Comment:
> nit: not sure if index.mustache is a good name for this, could be confusing in the future

It was originally named `email.mustache` and I worried that having a different template name to the view (index.js) might be confusing. Think `email.mustache` is a better name? Think we should rename them both to `email.js`?